spec/lib/main_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './spec/spec_helper'
+
+describe 'lib/main' do
+  describe 'stock_picker' do
+    let(:prices) { [17, 3, 6, 9, 15, 8, 6, 1, 10] }
+
+    it 'returns an array of size 2' do
+      stock_picker(prices).size.should == 2
```

Comment:
I think RSpec discourages the use of `should`, and recommends using `expect` instead. Looks like `should` is just an old syntax they still support for compatibility
https://github.com/rspec/rspec-expectations/blob/main/Should.md

Reply:
Done!